pkg/framework/proxies.go

```diff
@@ -0,0 +1,196 @@
+package framework
+
+import (
+	"context"
+	"errors"
```

Comment:
The linter will probably complain at you for not having a blank line between standard library packages and non-standard library packages

Reply:
updated